newIDE/app/src/ProjectManager/index.js

```diff
@@ -87,39 +102,293 @@ const styles = {
     padding: '0 8px 12px 12px',
```

Comment:
I think you can reduce the `marginTop` here, the space between the search bar and the tree view feels too big